docs/getting-started/summaryofchanges.md

```diff
@@ -41,8 +41,28 @@ The following features and enhancements were added.
 
 #### API Mediation Layer
 
+#### ZSS
+
+- Added zssServer.sh, configure.sh, and start.sh in packaging [#177](https://github.com/zowe/zss/pull/177)
+- Read JWT token information from environment variables if they exist in order to further support SSO during a standard install [#178](https://github.com/zowe/zss/pull/178)
+- In previous versions, ZIS did not use the version information provided in zss/version.txt. In this version, the ZIS build uses version.txt the same way ZSS uses it [#184](https://github.com/zowe/zss/pull/184)
+- A new control flag has been added. This flag allows users to enable the LPA dev mode without recompiling the module. In the LPA dev mode, the ZWESIS01 module will always be refreshed at start up and removed from LPA upon termination. Since removing modules from LPA is considered dangerous, a new warning message will be issued to the SYSLOG and SYSPRINT if the new mode is enabled [#140](https://github.com/zowe/zowe-common-c/pull/140)
+- New config options to enable the dev mode and LPA dev mode have been added. The dev mode only includes the LPA dev mode at the moment [#185](https://github.com/zowe/zss/pull/185/commits)
+
 #### Zowe App Server
 
+- Environment variables that contain PASSWORD or SECRET are no longer printed at startup [#120](https://github.com/zowe/zlux-app-server/pull/120)
```

Comment:
I am not sure if this should be in the release notes, because it indicates that prior these would have been printed. On the other hand, this tells people that upgrading is important.